src/trio/_core/_exceptions.py

```diff
@@ -63,6 +67,9 @@ class Cancelled(BaseException, metaclass=NoPublicConstructor):
     def __str__(self) -> str:
         return "Cancelled"
 
+    def __reduce__(self) -> tuple[typing.Callable[[], Cancelled], tuple[()]]:
```

Comment:
Any reason not to used `collections.abc.Callable`?

Reply:
No, just me forgetting to.